### ISSUE TITLE
Normalizacion

### ISSUE THREAD

**Issue by angelescox** [Exploratorio-DCC-PUC/Syllabus#242]
Tenemos que crear un dataframe columna por columna normalizada? o hay un método para reemplazar todo de una vez? que he estado buscando mucho y no me funciona... es para evitar hacer un hardcoding

**Comment by IchottMano** [comment]
Hola!

Si no me equivoco el método [normalize](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.normalize.html) recibe una matriz (dataframe) y lo normaliza completo sin necesidad de ir columna por columna.

Saludos!
